resolwe/flow/models/history.py

```diff
@@ -0,0 +1,193 @@
+"""Resolwe history model.
+
+It is used to track changes in objects.
+"""
+
+from typing import Any, Iterable, Type
+
+from django.contrib.postgres.fields import DateTimeRangeField
+from django.db import models
+from django.db.backends.postgresql.psycopg_any import DateTimeTZRange
+
+from .base import NAME_LENGTH, SLUG_LENGTH
+from .collection import Collection
+from .data import Data
+from .history_manager import HistoryMixin, history_manager
+
+
+class BaseHistory(models.Model):
```

Comment:
Mark this class as abstract: https://docs.djangoproject.com/en/4.2/topics/db/models/#abstract-base-classes

Reply:
This is not possible: I have a reference to this class from `SlugHistory` for example. This enables me to use the same class to track history of multiple objects.